reflectable/README.md

```diff
@@ -123,98 +113,52 @@ main() {
-transformer in their `pubspec.yaml`.
-
-It is possible to use globbing with entry points, based on the package
-[glob](https://pub.dartlang.org/packages/glob). For instance, `entry_points:
-["test/*_test.dart"]` can be used to specify that all Dart source files in
-`test` whose name ends in `_test.dart` are entry points.
-
-Now run `pub build --mode=debug web` to perform the transformation. This will
-rename the file `web/main.dart` and generate a new file `build/web/main.dart`.
-That file contains the data needed for  reflection, and a main function that
-will initialize the reflection framework before running the original `main`.
-When you run this file, it is important that the package-root is set to
-`build/web/packages`, because the reflectable package itself is transformed
-to a version that uses the generated data, instead of using `dart:mirrors`.
-
-Some elements in this scenario are optional: In the `pub build..` command, you
-could omit the final argument when it is equal to `web`, because that is the
-default. You could also omit `--mode=debug` if you do not wish to inspect or
-use the generated Dart code (typically, that would be because you only need the
-final JavaScript output). In 'pubspec.yaml', the `formatted` option can be
-omitted, in which case the output from the transformer will skip the formatting
-step (saving time and space, but putting most of the generated code into one
-very long line). Finally, in 'pubspec.yaml' you can use one more option:
-'suppress_warnings'; e.g., you could specify
-'suppress_warnings: missing_entry_point' in order to suppress the warnings
-given in the situation where an entry point has been specified in
-'pubspec.yaml', but no corresponding library (aka 'asset') has been provided
-by `pub` to the transformer.
-
-If you want to use the transformer directly rather than having `pub` invoke it,
-you can invoke it as a stand-alone tool. This is particularly relevant if you
-have several entry points, but at some point you only want to work with one of
-them, e.g., because it is being extended or debugged.
-
-The stand-alone version of the transformer will transform exactly the entry
-points you specify on the command line, no matter whether each of them is
-declared to be an entry point in 'pubspec.yaml'. It works in a similar way as
-`pub serve` because it is able to transform subsets of the entry points, but
-it differs in that it allows for inspection of the generated code, and it is
-completely under the programmer's control.
-
-The stand-alone version of the reflectable transformer is
-`bin/reflectable_transformer.dart`. It can be invoked using
-`pub global run` (after activating it with
-`pub global activate reflectable`):
+You may also wish to specify constraints on the version, depending on the
+approach to version management that your software otherwise employs.
 
+Please make sure that the reflectable package has been made active:
```

Comment:
Note that while it is totally valid to provide this pre-wrapped executable, we also want to support people running multiple `BuildAction`s with the same build script, so its probably worth exposing the reflectable builder directly and showing an example of how to use it (you could just point at `bin/reflectable_builder.dart` even).